src/Apps/Search/Components/NavigationTabs.tsx

```diff
@@ -30,7 +44,14 @@ export class NavigationTabs extends React.Component<Props> {
     const { exact } = options
 
     return (
-      <RouteTab to={to} exact={exact} preload={false}>
+      <RouteTab
+        to={to}
+        exact={exact}
+        preload={false}
+        onClick={() => {
```

Comment:
For some reason, with `preload={false}` this doesn't ever fire (a `debugger` here isn't triggered). Removing `preload={false}`, and this _does_ trigger, but somehow the context is lost and `this` is off. I'm assuming this is probably mid-transition and once the dust settles can revisit.